### PR TITLE
add cached fallback

### PR DESCRIPTION
**Pull request by SamVerschueren**
Did the same as with `viewport-list` but kept the live mechanism because this might change fairly quick if a new device (for instance a new iPhone) is released. I can always drop that as well if you guys want that. If we aren't dropping that, I will extract the `got` call and parsing to an utility file which can be reused by both files.

Also it seems that the `countries` data is not present anymore (https://www.w3counter.com/globalstats.php), hence the `.failing` test. Should we drop it?

--------------

Will also :lipstick: and ES2015ify in a follow up PR in order to keep this one fairly clean.

// @sindresorhus

### CODE REVIEW

**Comment by sindresorhus** [comment]
LGTM

> Also it seems that the countries data is not present anymore (https://www.w3counter.com/globalstats.php), hence the .failing test. Should we drop it?

Yup

File: index.js [hunk 1]
```diff
@@ -1,39 +1,26 @@
 'use strict';
-var cheerio = require('cheerio');
-var got = require('got');
 var Promise = require('pinkie-promise');
+var utils = require('./lib/utils');
+var stats = require('./data.json').stats;
```
Comment (SamVerschueren) [inline index.js]: Should I rename this to `cachedStats` or something? Because below I use `stats` as well. This might be confusing.

**Comment by SamVerschueren** [comment]
Extracted it to a `lib/utils` file. Not sure about the name of the file though :).

File: index.js (hunk 1 above)
Comment (sindresorhus) [inline index.js]: 👍

**Comment by sindresorhus** [comment]
LGTM :)

**Pull request merged**